Automattic/woocommerce-payments
# Cleanup explat wcpay_empty_state_preview_mode_v5

Fixes https://github.com/woocommerce/team-ghidorah/issues/255



#### Changes proposed in this Pull Request

This PR cleans up changes made for `wcpay_empty_state_preview_mode_v5` experiment.









#### Testing instructions

1. Checkout this branch and run `npm run start` to build the changes.
2. Test `Deposits` and `Transactions` pages and confirm  they work as expected.
3. Open your browser inspector and go to `Network` tab.
4. Search for `wcpay_empty_state_preview_mode_v5` and refresh the page
5. Confirm there's no request for `wcpay_empty_state_preview_mode_v5`






*

-------------------


**Post merge**
